test/integration/pagination_test.go

```diff
@@ -19,10 +19,11 @@ import (
 	"github.com/stretchr/testify/assert"
 	authenticationv1 "k8s.io/api/authentication/v1"
 	corev1 "k8s.io/api/core/v1"
-	errs "k8s.io/apimachinery/pkg/api/errors"
 	rbacv1 "k8s.io/api/rbac/v1"
+	errs "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
This change looks unnecessary.

Reply:
this is the result of running `goimports` on the file after saving the changes I made elsewhere. We agreed at the kubearchive go code should be formatted with `goimports`

Reply:
How did it ever get committed. Don't we always run `goimports`?